styleguide/components/00-mixins/general/_mixins.scss

```diff
@@ -276,18 +214,50 @@
   @return if($minus, -$result, $result);
 }
 
+///
+/// Defines small text.
+///
+/// @since 3.0.0
+/// @group tools
+/// @access private
+/// @author Gert-Jan Meire
+///
 @mixin small-text {
   font-size: .6rem;
 }
 
+///
+/// Defines medium text.
```

Comment:
Should be obsolete.
If medium text is not small or large: it's default.

Reply:
As discussed to be changed later.